### PR TITLE
Update data collected for audio sets

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Addresses https://github.com/WordPress/openverse-api/issues/157#issuecomment-911539017 by @dhruvkb

## Description

This PR updates data collected for audio sets during the crawl process. This is to populate missing fields and update their names to match the API database models.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Additionally tagging @obulat and @krysal for review as they have worked with these scripts before.

File: README.md [hunk 1]
```diff
@@ -175,7 +175,12 @@ To follow the logs of the running container:
 just logs
 ```
 
-To see the Airflow web UI, point your browser to `localhost:9090`.
+To see the Airflow web UI, point your browser to `localhost:9090`. To log in, you will need to set up an account.
+
+```shell
+cd openverse_catalog
+docker-compose exec webserver airflow users create -u <username> -p <password> -f <first_name> -l <last_name> -e <email>
```
Comment (dhruvkb) [inline README.md:182]: Should this be a `just` command? I don't see it being used frequently but it has a lot of required arguments that could have default values when setting up in `just`.

Comment (dhruvkb) [reply README.md]: Updated the instructions in 59455b7.

**Review by zackkrida — APPROVED** [review]

**Review by obulat — COMMENTED** [review]
The changes work great for the data collection. However, the collected data will not be stored in the database because the loader code is not updated. I'm working on refactoring the loader code so that it is easier to update data model both in data extraction and data storage (Getting data from the API and saving in to the TSV file - saving the data from TSV to the database)

**Comment by dhruvkb** [comment]
@obulat let me know if there's something I can do that does not conflict with your planned changes to the loader.

**Comment by zackkrida** [comment]
Replaced by #193, which is sadly less granular but fixes many issues with merge order and our long-term fix for column order.

**Pull request closed without merging**